web/src/portico/team.ts

```diff
@@ -122,21 +146,31 @@ export default function render_tabs(contributors: Contributor[]): void {
         }),
     );
 
-    for (const repo_name of all_repository_names) {
-        const tab_name = repo_name_to_tab_name[repo_name];
-        if (!tab_name) {
+    for (const tab_name of all_tab_names) {
+        const repo_list = tab_name_to_repo_list[tab_name];
+        if (!repo_list) {
             continue;
         }
         // Set as the loading template for now, and load when clicked.
         $(`#tab-${CSS.escape(tab_name)} .contributors-grid`).html($("#loading-template").html());
 
         $(`#${CSS.escape(tab_name)}`).on("click", () => {
-            if (!loaded_repos.includes(repo_name)) {
-                const filtered_by_repo = contributors_list.filter((c) => c[repo_name] ?? 0);
-                const html = filtered_by_repo
+            if (!loaded_tabs.includes(tab_name)) {
+                const filtered_by_tab = contributors_list.filter((c) =>
+                    repo_list.some((repo_name: RepositoryName) => c[repo_name] !== undefined),
+                );
+                const html = filtered_by_tab
                     .sort((a, b) => {
-                        const a_commits = a[repo_name] ?? 0;
-                        const b_commits = b[repo_name] ?? 0;
+                        const a_commits = repo_list.reduce(
+                            (commits: number, repo_name: RepositoryName) =>
+                                commits + (a[repo_name] ?? 0),
+                            0,
+                        );
+                        const b_commits = repo_list.reduce(
+                            (commits: number, repo_name: RepositoryName) =>
+                                commits + (b[repo_name] ?? 0),
+                            0,
+                        );
```

Comment:
This sort function seems weird; we should just be sorting tab data objects that already have the total commits for the tab computed, not doing math in the sort function.